Provide exit code at the end of chkrootkit script, thanks to Arjan Opmeer (Closes: #541060)
--- a/chkrootkit
+++ b/chkrootkit
@@ -2886,5 +2886,5 @@ do
 
     fi
 done
-
+exit 0
 ### chkrootkit ends here.
